content/showcase/_index.html

```diff
@@ -197,6 +197,14 @@ <h1 class="title is-2">UQCS Showcase</h1>
                         link="https://github.com/sktapsall/CheckerBOT"
                         class="showcase-element has-text-centered"
                     >}}
+                    {{< showcase-item
+                        src="/img/media/showcase/nie.png"
+                        title="UQCS2021-Team-NIE"
+                        caption="Hackathon the Experience. Wanted to make a game based on caring for hackers needs."
```

Comment:
This could probably use a little more detail.

Reply:
Added additional detail.